kyo-scheduler/jvm-native/src/test/scala/kyo/scheduler/SchedulerTest.scala

```diff
@@ -80,47 +80,6 @@ class SchedulerTest extends AnyFreeSpec with NonImplicitAssertions {
         }
     }
 
-    "cycle" - {
```

Comment:
It's not easy to test this at the scheduler level because, even if we submit more tasks, the concurrency regulator can introduce new workers. The behavior is covered in `WorkerTest` so I'm removing the tests here.